lib/pure/ospaths.nim

```diff
@@ -423,12 +423,22 @@ proc isAbsolute*(path: string): bool {.rtl, noSideEffect, extern: "nos$1".} =
   ## Checks whether a given `path` is absolute.
   ##
   ## On Windows, network paths are considered absolute too.
+  runnableExamples:
+    doAssert "".isAbsolute.not
```

Comment:
``.not`` is not a style we use anywhere else.

Reply:
that was a workaround after hitting that bug: https://github.com/nim-lang/Nim/issues/8300 ; but I've fixed it to `doAssert(not condition)`